.github/workflows/start-beta.yml

```diff
@@ -43,7 +46,7 @@ jobs:
 
       - name: Start Release
         env:
-          GH_TOKEN: ${{ github.token }}
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Are those equivalent? Also should this also be changed in start-ga?

Reply:
Yes, those are the same - there's no firm rule when to use which one, so it's a matter of personal preference. This was an accidental change though - I was originally changing it to the app token and when I realized that's not needed, I changed it "back", but must have not realized I'm using the secrets syntax. Will revert.

Reply:
Got it, thanks for the detailed explanation!